src/service/viewer-impl.js

```diff
@@ -263,6 +264,12 @@ export class Viewer {
         || this.isWebviewEmbedded_
         || !ampdoc.isSingleDoc());
 
+    /**
+     * Whether the AMP document is embedded in a Chrome Custom Tab.
+     * @private @const {boolean}
+     */
+    this.isCustomTab_ = this.win.location.search.indexOf('amp_agsa=1') != -1;
```

Comment:
Might also want to sanity check that `this.isIframed_` or even `this.isEmbedded_` == true to avoid this behavior in other contexts.

Reply:
I believe that in CCT, the document will be top level (i.e., it'll be as though it's loading in its own dedicated Chrome). In this case, should I add a check that we're *not* iFramed_ for added confidence?